frontend/components/Footer.js

```diff
@@ -6,7 +6,7 @@ const Footer = () => (
     <hr className="footer_hr" />
     <div className="container">
       <div className="row">
-        <div className="col-lg-8 col-lg-offset-2 text-center">
```

Comment:
What does this do?

Reply:
It looked like it was to make the element be sized the right way, but I didn't see a reason why it was done this way instead of just saying `width: 100%`.